wagtail_localize_rws_languagecloud/templates/wagtail_localize_rws_languagecloud/admin/update_translations_override.html

```diff
@@ -0,0 +1,23 @@
+{% extends "wagtailadmin/base.html" %}
+
+{% load i18n %}
+
+{% block titletag %}{{ view.get_title }} {{ view.get_subtitle }}{% endblock %}
+
+{% block content %}
+    {% include "wagtailadmin/shared/header.html" with title=view.get_title subtitle=view.get_subtitle icon="doc-empty-inverse" %}
+
+    <div class="nice-padding">
+      <div class="help-block help-info">
+        <p><strong>{% trans "Translations for this page can't be updated right now" %}</strong></p>
+        <p>
+          {% trans "LanguageCloud is currently translating this page. This means that translated pages cannot be synced at this time in order to avoid conflicts." %}
+        </p>
+        <p>
+          {% trans "Translated pages can be resynced as soon as all of the ongoing translations are completed." %}
```

Comment:
Minor point, but the wording is a bit off here. 2 points:

1. LanguageCloud isn't translating the object - their (human) translators are. LanguageCloud is the software they are using.
2. Presumably this also applies to snippets - the entity is not always a page.

Reply:
Oh, I completely misunderstood that then. I thought LanguageCloud was automated. Does that mean there are people reading my test page entries? 😱 😂 

I will update the copy.

Reply:
I've update this copy now:

> This page is currently being translated on LanguageCloud. It cannot be synced at this time in order to avoid conflicts with existing translated pages.
> 
> Translated pages can be resynced as soon as all of the ongoing translations are completed.